.travis.yml

```diff
@@ -1,31 +1,28 @@
 language: java
 jdk: openjdk6
+env:
+- DSPACE_DIR=/opt/dryad/test
 cache:
     directories:
     - $HOME/.m2
 before_install:
 - ./test/bin/travis-install-maven.sh
-- psql -U postgres -c "create user dryad_test_user with createdb;"
-- createdb -U dryad_test_user dryad_test
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/database_schema.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/database_create_doi_table.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/database_change_payment_system.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/database_terms_and_condition.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/collection-workflow-changes.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/atmire-versioning-changes.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/database_dryad_groups_collections.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/database_dryad_registries.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/dryad-rest-webapp.sql
-- psql dryad_test -U dryad_test_user < ./test/etc/postgres/test-system-curator.sql
-- psql dryad_test -U dryad_test_user < ./dspace/etc/postgres/update-sequences.sql
-- psql dryad_test -U dryad_test_user < ./test/etc/postgres/test-journal-landing.sql
 - export PATH="$TRAVIS_BUILD_DIR/apache-maven-2.2.1/bin:$PATH"
+- export DRYAD_CODE_DIR=$TRAVIS_BUILD_DIR
+- export DRYAD_TEST_DIR=$DSPACE_DIR
+- export PGUSER=dryad_test_user
+- export PGDATABASE=dryad_test_db
+- export PGPASSWORD=
+- export PGHOST=127.0.0.1
+- export PGPORT=5432
+
 install:
 - ./test/bin/travis-install-missing-poms.sh
 - ./test/bin/travis-install-dryad-dir.sh
-script: mvn package -DskipTests=false -Ddefault.dspace.dir=/opt/dryad-test
+script: mvn package -DskipTests=false -Ddefault.dspace.dir=$DSPACE_DIR
 after_failure: find . -type f -path "*/surefire-reports/*" -name "*.txt" -print -exec cat {} \;
 notifications:
+  email: false
   slack:
     secure: EqYW7e3FhVxoZzVS4FR52ajePoDbyeOsYcXA9b2BjUWDNc5WNFxRDwNfkxmD09BOAI1g7fKH+Fj5VrtnsUwh7kYosNBj9anjl9CleJ8xGQR35avoaXMJQ3BcULIBfSsUUEwaT+M8uHnM1PNC9DW1uq5w+Hd/9wASwJR5g5BU49I=
```

Comment:
I don't think so? That line has been there for over a year and I don't think we've seen anything like that.